vars/withPodmanRemoteContext.groovy

```diff
@@ -0,0 +1,25 @@
+//Run podman remote build and push
+// Available parameters:
+//    remotehost:              string -- OCP secret name containing the remote host
+//    remoteuid:               string -- OCP secret name containing the remote user uid
+//    sshkey:                  string -- OCP secret name containing the sshkey
```

Comment:
Some minor tweaks, but also parameter renames for consistency with other functions in this shared lib:

```suggestion
//    remoteHost:              string -- OCP secret name containing the remote host
//    remoteUid:               string -- OCP secret name containing the remote user UID
//    sshKey:                  string -- OCP secret name containing the SSH key
```

Reply:
Good point. The fact that it's likely backed by a k8s secret is an implementation detail hidden from coreos-ci-lib. The most correct would be to say "Credential ID for (remote host|remote user UID|SSH key)", since that's how we use it.